Assets/Scripts/Player/Movement/PlayerMovement.cs

```diff
@@ -3,34 +3,63 @@
 using FrameWork.Extensions;
 using FrameWork.GridSystem;
 using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Serialization;
 
 namespace Player.Movement
 {
     public class PlayerMovement : MonoBehaviour
```

Comment:
```suggestion
    public sealed class PlayerMovement : MonoBehaviour
```
Dacht dat deze al gedaan was, zou je kunnen fixen :D